### PR TITLE
Add molbloom package

### PR DESCRIPTION
**Pull request by hmacdope**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by hmacdope** [comment]
@whitead are you happy to be a maintainer for this recipie?

**Comment by whitead** [comment]
If it is possible to have two, please add yourself first and me second. If not, then you may add me alone.

**Comment by hmacdope** [comment]
You can have as many as you like, I will switch the order to the one you propose :)

**Comment by hmacdope** [comment]
@ijpulidos are you happy to be a maintainer for this recipie?

**Comment by ijpulidos** [comment]
@hmacdope Yes, happy to be a maintainer for this recipe as well. Thanks!

**Comment by hmacdope** [comment]
@conda-forge/help-python-c could I grab a review here please?

File: recipes/molbloom/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/whitead/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 4fb187a612f8da59473ac78b893d94fb6c1c6c2daf4027a8228e8b9dc00e7b13
+
+build:
+  noarch: python
```
Comment (xhochy) [inline recipes/molbloom/meta.yaml:13]: If you have compiled code, you cannot be `noarch`.
```suggestion
```

File: recipes/molbloom/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  host:
+    - python >=3.6
+    - pip
+    - cython
+  run:
+    - python >=3.6
+    - rdkit
+    - importlib_resources
+
+test:
```
Comment (xhochy) [inline recipes/molbloom/meta.yaml:30]:
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
```

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/molbloom/meta.yaml (hunk 2 above)
Comment (hmacdope) [reply recipes/molbloom/meta.yaml:29]: @xhochy pip check doesn't seem to work due to the way compiled extensions are imported  within the package, are we able to skip this bit?

Comment (xhochy) [reply recipes/molbloom/meta.yaml:29]: `pip` check doesn't execute the python code at all but simply checks metadata consistency. So imports shouldn't affect it.

Comment (hmacdope) [reply recipes/molbloom/meta.yaml:29]: Lets try, perhaps you can see what is going on, bit bamboozling to me.

**Comment by hmacdope** [comment]
Well there  you go, seems to work.

**Comment by hmacdope** [comment]
Wow it worked! Thanks @xhochy

**Comment by hmacdope** [comment]
@xhochy could I grab a re-review please?

File: recipes/molbloom/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  requires:
+    - pip
+  commands:
+    - pip check
+  requires:
+    - pip
+  commands:
+    - pip check
```
Comment (xhochy) [inline recipes/molbloom/meta.yaml:37]: This is now duplicated and one of the instances should be removed.
```suggestion
```

**Review by xhochy — CHANGES_REQUESTED** [review]

**Comment by hmacdope** [comment]
@xhochy apologies, should have picked that up

**Review by xhochy — APPROVED** [review]

**Pull request merged**